govcd/uploadtask.go

```diff
@@ -0,0 +1,33 @@
+package govcd
+
+import (
+	"fmt"
+	"time"
+)
+
+type UploadTask struct {
+	uploadProgress *float64
+	*Task
+}
+
+func NewUploadTask(task *Task, uploadProgress *float64) *UploadTask {
```

Comment:
Please add a comment to document public APIs in this file.

Reply:
Added.